{"uuid": "64e7488d-d8be-4327-8c6a-f6641e986911", "counter": 7, "stdout": "\u001b[1;31mfatal: [2001:400:a100:3020:f816:3eff:fe91:8064]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: Can't open user config file FILE: No such file or directory\", \"unreachable\": true}\u001b[0m", "start_line": 6, "end_line": 7, "runner_ident": "99161b90-5ee5-4da9-b580-ce6f1d58404c", "event": "runner_on_unreachable", "pid": 41934, "created": "2025-01-17T23:29:54.236223", "parent_uuid": "e6082347-933e-cab2-19cc-00000000000c", "event_data": {"playbook": "/Users/sanjanadas/fabfed_venv/lib/python3.10/site-packages/fabfed/provider/janus/ansible/janus.yml", "playbook_uuid": "b22ab2a9-1e77-4aa1-a574-19953f0ee863", "play": "Ansible Service Setup", "play_uuid": "e6082347-933e-cab2-19cc-000000000002", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "e6082347-933e-cab2-19cc-00000000000c", "task_action": "gather_facts", "resolved_action": "ansible.builtin.gather_facts", "task_args": "", "task_path": "/Users/sanjanadas/fabfed_venv/lib/python3.10/site-packages/fabfed/provider/janus/ansible/janus.yml:2", "host": "2001:400:a100:3020:f816:3eff:fe91:8064", "remote_addr": "2001:400:a100:3020:f816:3eff:fe91:8064", "start": "2025-01-17T23:29:54.222699", "end": "2025-01-17T23:29:54.235987", "duration": 0.013288, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: Can't open user config file FILE: No such file or directory", "changed": false}, "uuid": "64e7488d-d8be-4327-8c6a-f6641e986911"}}